src/main/java/core/basesyntax/FigureProducer.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureProducer {
+    public static final int MAGIC_NUMBER = 5;
```

Comment:
```suggestion
    private static final int FIGURES_QUANTITY = 5;
```
if we don't use variable anywhere else let's make private 
[magic number definition](https://help.semmle.com/wiki/display/JAVA/Magic+numbers%3A+use+defined+constant#:~:text=A%20magic%20number%20is%20a,for%20other%20programmers%20to%20understand.)